datafusion/src/logical_plan/plan.rs

```diff
@@ -57,6 +57,47 @@ pub enum JoinConstraint {
     Using,
 }
 
+/// Evaluates an arbitrary list of expressions (essentially a
+/// SELECT with an expression list) on its input.
+#[derive(Clone)]
+pub struct ProjectionPlan {
+    /// The list of expressions
+    pub expr: Vec<Expr>,
+    /// The incoming logical plan
+    pub input: Arc<LogicalPlan>,
+    /// The schema description of the output
+    pub schema: DFSchemaRef,
+    /// Projection output relation alias
+    pub alias: Option<String>,
+}
+
+/// Filters rows from its input that do not match an
+/// expression (essentially a WHERE clause with a predicate
+/// expression).
+///
+/// Semantically, `<predicate>` is evaluated for each row of the input;
+/// If the value of `<predicate>` is true, the input row is passed to
+/// the output. If the value of `<predicate>` is false, the row is
+/// discarded.
+#[derive(Clone)]
+pub struct FilterPlan {
+    /// The predicate expression, which must have Boolean type.
+    pub predicate: Expr,
+    /// The incoming logical plan
+    pub input: Arc<LogicalPlan>,
+}
+
+/// Window its input based on a set of window spec and window function (e.g. SUM or RANK)
+#[derive(Clone)]
+pub struct WindowPlan {
```

Comment:
Thank you @ic4y 

I feel like these names are not consistent with the other structs in this file

How about renaming 
```
ProjectionPlan --> Projection
FilterPlan --> Filter
WindowPlan --> Window
```

To be consistent with the naming of  the model of  `CrossJoin` (not `CrossJoinPlan`), `Repartition` (not `RepartitonPlan`), etc)?

Reply:
OK, thanks @alamb  I will modify according to your suggestion.